source/Octopus.Tentacle/Commands/PollCommand.cs

```diff
@@ -45,11 +46,14 @@ public class PollCommand : AbstractStandardCommand
             this.log = log;
             this.selector = selector;
 
-            api = AddOptionSet(new ApiEndpointOptions(Options));
+            // purposefully not adding this with AddOptionSet because
+            // we only want to validate it if reuseThumbprint is false.
+            api = new ApiEndpointOptions(Options);
```

Comment:
Understandable why this was done but it is hard to follow. If only validation was a separate method within `AbstractCommand` :(

Reply:
Yeah it's dumb because that's all that is done when an OptionSet is added via `AddOptionSet`.